src/main/java/com/entropyteam/entropay/employees/dtos/EmployeeDto.java

```diff
@@ -40,6 +40,8 @@ public record EmployeeDto(UUID id,
                           String healthInsurance,
                           List<PaymentInformationDto> paymentInformation,
                           List<UUID> technologies,
+                          @Email
+                          String labourEmail,
```

Comment:
What happen on null with this validation?

Reply:
@Email annotation only checks if the input has an email expression, it can be null. if dont have an email structure it drops an message error "must be a well-formed email address"